config/django_config/base.py

```
@@ -6,6 +6,7 @@
 """
 
 from config.env import BASE_DIR, env
+from config.django_config.logging import LOGGING
```

Ruff (0.8.2)

`config.django_config.logging.LOGGING` imported but unused

Remove unused import: `config.django_config.logging.LOGGING`

(F401)